Sources/Authenticator/Authenticator.swift

```diff
@@ -89,6 +96,7 @@ public struct Authenticator<LoadingContent: View,
     /// - Parameter content: The content associated with the ``AuthenticatorStep/signedIn`` step, i.e. once the user has successfully authenticated.
     public init(
         initialStep: AuthenticatorInitialStep = .signIn,
+        totpOptions: TOTPOptions? = nil,
```

Comment:
Have you synced with Android on this name, to be consistent?

Also, I don't really have the full context so I could totally be wrong, but isn't this value only applicable to the `ContinueSignInWithTOTPSetupView` view, and not the Authenticator itself?

If that's the case, I suggest you go with a view modifier-like function instead, similar to what we do for `hidesSignUpButton(_:)` (only applicable to the SignInView) and `signUpFields(_:)` (Only applicable to SignUpView)